velox/dwio/parquet/tests/duckdb_reader/ParquetTpchTest.cpp

```diff
@@ -156,7 +156,26 @@ std::unordered_map<std::string, std::string>
             R"(COPY (SELECT c_custkey, c_name, c_address, c_nationkey, c_phone,
         c_acctbal::DOUBLE as c_acctbal, c_mktsegment, c_comment
         FROM {}) TO '{}' (FORMAT 'parquet', ROW_GROUP_SIZE {}))"),
-};
+        std::make_pair(
+            "nation",
+            R"(COPY (SELECT n_nationkey, n_name, n_regionkey, n_comment
```

Comment:
`select *`  should work since we do not have to cast any columns for these tables.